src/components/Checkbox.js

```diff
@@ -22,22 +22,21 @@ const defaultProps = {
 
 const Checkbox = ({
     isChecked,
-    onCheckboxClick,
+    onClick,
     label,
 }) => (
     <View style={styles.flexRow}>
-        <Pressable onPress={() => onCheckboxClick(!isChecked)}>
-            <View style={[styles.checkboxContainer, isChecked && styles.checkedContainer]}>
-                <Icon src={Checkmark} fill="white" height={14} width={14} />
-            </View>
-        </Pressable>
-        {label && (
-            <Pressable onPress={() => onCheckboxClick(!isChecked)}>
+        <Pressable onPress={() => onClick(!isChecked)}>
+            {label ? (
```

Comment:
are we hiding the checkbox from now on if there's a label present? I think we should show the Checkmark icon regardless, otherwise, it's hard to know that it's a clickable element

Reply:
Ah, good catch. I think I read the original code wrong. I need to revert this.